include/configs/sun50iw1p1.h

```diff
@@ -352,23 +352,23 @@
 	"initrd_filename=initrd.img\0" \
 	"bootenv_filename=uEnv.txt\0" \
 	"load_bootenv=" \
-		"fatload mmc ${boot_part} ${load_addr} ${bootenv_filename}\0" \
+		"for prefix in / /boot/; do if test -e mmc ${boot_part} ${bootenv_filename}; then load mmc ${boot_part} ${load_addr} ${prefix}${bootenv_filename}; fi; done\0" \
```

Comment:
I think having a hierarchy of / and then /boot makes more sense, but I don't really have a strong opinion on this.